test/addon_runners/chrome/data/panel.js

```diff
@@ -1,3 +1,4 @@
+/* jshint ignore: start */
```

Comment:
if you're ignoring this directing is the ignore comment still needed? Either way, sounds fine.